src/i18n/ja.yml

```diff
@@ -0,0 +1,135 @@
+i18n:
+  lang:
+    code: ja
+    display: 日本語
+    ja: true
+  nav:
+    home: ホーム
+    wallets: ウォレット
+    quickstart: クイックスタート
+    faqs: よくある質問
+  footer:
+    copyright: Copyright &copy; 2017-2019 CashShuffle. All rights reserved.
+    donate:
+      instructions: 私たちはBitcoin Cashにプライバシーとファンジビリティをもたらすため、常に努力しています。皆様から寄付を頂ければ幸いに存じます。
+    client:
+      title: クライアントソフトウェア
+      description: CashShuffleのリファレンス実装は、人気のElectron Cashウォレットから利用可能です。
+    server:
+      title: サーバーソフトウェア
+      description: サーバーサイドソフトウェアはGolangで書かれておりGitHubで利用可能です。
+    docs:
+      title: ドキュメンテーション
+      description: 開発者および愛好家のための技術文書
+  home:
+    title: CashShuffle  -  Bitcoin Cashのためのプライバシー機能
```

Comment:
NIT: Don't need 2 spaces here. =)

Reply:
Checking with translator. Also I fixed a bunch of whitespace at the end of lines.

Reply:
Ok I fixed all the whitespace at the end of lines and also removed those double spaces you found. Translator did not intend that.

dbe24e2

Reply:
I pushed to the wrong branch. Hold on.

Reply:
4ef041c

Reply:
Also in the original english: 592c8ff